optaweb-vehicle-routing-frontend/cypress/integration/fromLocationsToRoute.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe('Locations can be added and route is computed', () => {
+
+  /**
+   * Adds a location by searching for a city of a given name.
+   * @param {city name} name 
+   */
+  const addCity = (name) => {
+    cy.get('[data-cy=demo-location-text-input]').type(name);
+    cy.get('[data-cy=demo-location-item-button-1]').click();
+  };
+
+  /**
+   * Clears locations by clicking on a 'Clear' button.
+   */ 
+  const clearLocations = () => {
+    // Add one city to make sure there is a location in the list and the clear button shows up
+    addCity('Brussels');
+    cy.get('[data-cy=demo-clear-button]').click({ force: true });
+  }
```

Comment:
We should configure ESLint to check Cypress tests too to keep the code style consistent (missing semicolon here).

I can do that after the PR is merged unless you want to give it a shot (I can help).

Reply:
I will gladly take a look - I just ran the eslint manually a couple of minutes ago and output proved we need it there.

Reply:
You'll need this: https://github.com/cypress-io/eslint-plugin-cypress to recognize the `cy` global variable, adjust eslint command to run in the root dir, not only src (`"lint": "./node_modules/eslint/bin/eslint.js --ext .js,.ts,.tsx ."` works for me) and add [`.eslintignore`](https://eslint.org/docs/user-guide/configuring#ignoring-files-and-directories) to exclude some directories that we don't want to check:
```
/build
/coverage
/local
/node
/public
/target
```

Reply:
+1 I have already stumbled upon that plugin.